index.js

```diff
@@ -5,12 +5,11 @@ const importer = require('./importer');
 
 const usage = `
 Usage:
-  csv2influx init        Creates template config file
-  csv2influx data.csv    Loads config file from current directory
-                         then imports file data.csv to your influx
+  csv2influx init                                       Creates template config file
+  csv2influx --config path/to/config.json data.csv      Loads config from path/to/config.json then imports file data.csv to your influx
```

Comment:
I proposed to make `--config` optional, so here would be great to add few words to help about it.